File: ProjetoPOO/Models/Servico.cs
using System;

namespace ProjetoPOO.Models
{
    public class Servico : ModeloId
    {
        public string Nome { get; set; }
        public string Descricao { get; set; }
        public float Preco { get; set; }

        public Servico() { }

        public Servico(int id, string nome, string descricao, float preco)
        {
            Id = id;
            Nome = nome;
            Descricao = descricao;
            Preco = preco;
        }

        public override string ToString()
        {
            return $"ID: {Id} | Nome: {Nome} | Descrição: {Descricao} | Preço: {Preco:C}";
        }
    }
}

File: ProjetoPOOPython/Models/servico.py
from modeloId import ModeloId

class Servico(ModeloId):
    def __init__(self, id: int = 0, nome: str = "", descricao: str = "", preco: float = 0.0):
        super().__init__(id)
        self.nome = nome
        self.descricao = descricao
        self.preco = preco

    def __str__(self):
        return f"ID: {self.id} | Nome: {self.nome} | Descrição: {self.descricao} | Preço: R$ {self.preco:.2f}"
